lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/v2x/GenericV2xMessage.java

```diff
@@ -82,4 +89,8 @@ public String toString() {
                 + ", encodedPayload=" + payload + '}';
     }
 
+    public GenericV2xMessage duplicate(MessageRouting routing) {
```

Comment:
Please add the override annotation. 
```suggestion
    @Override
    public GenericV2xMessage duplicate(MessageRouting routing) {
```